var/spack/repos/builtin/packages/elemental/package.py

```diff
@@ -84,9 +84,12 @@ class Elemental(CMakePackage):
     depends_on('metis +int64', when='+int64')
     depends_on('mpi')
     # Allow Elemental to build internally when using 8-byte ints
-    depends_on('scalapack', when='+scalapack ~int64_blas')
+    depends_on('scalapack', when='+scalapack blas=openblas ~int64_blas')
```

Comment:
> I did this because of an issue with intel-mkl getting included in an inconsistent fashion.  It is probably not the best solution, but until variants on virtual dependences are fixed it is all a bit hacky.

yes, it is hacky. Either way I am not in favour of this change as it prevents using other Blas/Lapack libs.

As for the MKL, it still work when you use `~openmp` blas.

Bottom line: I don't think issue with MKL justify this change.